functions/exercises.py

```diff
@@ -267,12 +310,24 @@ def get_pascal_rows(number):
     :return:
     """
     # INSERT YOUR CODE HERE
-    #
+    if number == 1:
+        return [[1]]
+    else:
+        res_array = get_pascal_rows(number - 1)
+        current_row = [1]
+        previous_row = res_arr[-1]
+        
+        for i in range(len(prev_row)-1):
+            current_row.append(prev_row[i] + prev_row[i +1 ])~

```

Comment:
```suggestion
            current_row.append(prev_row[i] + prev_row[i +1 ])
```
Is this passing in the tests?